subprojects/docs/src/docs/release/notes.md

```diff
@@ -100,7 +101,27 @@ gradle mAL:cT
 
 Note, that even though the kebab case name matching works with tasks too, the recommendation is still to use camel case for them. 
 
-To learn more about name abbreviation, check out the [user guide](userguide/command_line_interface.html#task_name_abbreviation).  
+To learn more about name abbreviation, check out the [user guide](userguide/command_line_interface.html#task_name_abbreviation).
+
+## Support for version ranges in repository content filtering
+
+With [repository content filtering](userguide/declaring_repositories.html#sec:repository-content-filtering), builds can control which repositories are queried for which dependency.
```

Comment:
How about the following slightly changed version?

Builds can control which repositories are queried for which dependency for better performance and security using [repository content filtering](userguide/declaring_repositories.html#sec:repository-content-filtering).

Reply:
Will push the changed version directly.

Reply:
Done